hotkeywrapper.cc

```diff
@@ -278,7 +278,7 @@ bool HotkeyWrapper::checkState(quint32 vk, quint32 mod)
 
       // Grab the second key, unless it's grabbed already
       // Note that we only grab the clipboard key only if
-      // the sequence didn't begin with it
+      // the sequence begins with it

```

Comment:
I wonder whether the comment actually refers to the key grabbing in another place and is in fact correct:
https://github.com/goldendict/goldendict/blob/83020389c802577075831f7b45cc1a4d3017c86f/hotkeywrapper.cc#L700-L701